### ISSUE TITLE
Any ways to contribute to this?

### ISSUE THREAD

**Issue by kroonhorstdino** [campbellcole/computercraft-rs#1]
This looks like an interesting crate, is there any chance to contribute in a meaningful way?

**Comment by campbellcole** [comment]
Absolutely! It's been a little while since I've touched this project but there are still some goals I have for it. The to-do list in the README is definitely a good place to start. Namely documentation and access to Lua globals (`os`, etc.). I haven't decided how exactly globals would work but if you have any ideas I'm open to them.

Also if there are any peripherals that you want to use this with, feel free to add wrappers for them. You can use the `wrappers` module and its items as a reference. There are several macros that make it trivial.
